src/main/java/service/impl/TransactionParserServiceImpl.java

```diff
@@ -0,0 +1,21 @@
+package service.impl;
+
+import exception.FruitShopException;
+import model.FruitTransaction;
+import service.TransactionParserService;
+
+public class TransactionParserServiceImpl implements TransactionParserService {
+    private static final int OPERATION = 0;
+    private static final int FRUIT = 1;
+    private static final int QUANTITY = 2;
+    private static final String SEPARATOR = ",";
+
```

Comment:
here, add method for parsing list of strings, extract this business logic from main